app.rambox.ramboxce.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>app.rambox.ramboxce</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>Rambox Community Edition</name>
+  <developer_name>Rambox</developer_name>
+  <url type="homepage">https://rambox.app/#home</url>
+  <summary>Put all your online Services in one Application!</summary>
+  <url type="homepage">https://rambox.app/</url>
+  <url type="github">https://github.com/ramboxapp/community-edition</url>
+  <description>
+    <p>Rambox gives you the possibility to add common services many times you need, all in one place. It is completely open source</p>
+    <p>This Program let you create seperate containers for different services and accounts in tabs and checks for notifications in the Background. This basically lets you convert as many websites as you want into desktop Application without having to Install </p>
+		<p>It's perfect for people who work with many services for business and private accounts. (unofficial Flatpak)</p>
+    <p>Disclosure: Rambox is not affiliated with any of the messaging apps offered.</p>
+    <p>(Note: You can buy a proprietary PRO version, but doesn't has a Flatpak yet)</p>
+  </description>
+  <content_rating type="oars-1.0">
+    <content_attribute id="social-chat">moderate</content_attribute>
+    <content_attribute id="social-audio">moderate</content_attribute>
+    <content_attribute id="social-contacts">intense</content_attribute>
+  </content_rating>
+  <screenshots>
+    <screenshot type="default">
+      <image>start.png</image>
+      <caption>Start Sceen</caption>
+    </screenshot>
+    <screenshot type="default">
+      <image>discord.png</image>
+      <caption>Discord</caption>
+    </screenshot>
+    <screenshot type="default">
+      <image>telegram_web.png</image>
+      <caption>Telegram Web</caption>
+    </screenshot>
+  </screenshots>
+  <launchable type="desktop-id">app.rambox.ramboxce.desktop</launchable>
```

Comment:
If you keep upstream's `rambox.desktop`, then use that here and it will be renamed automatically.

Reply:
but the upstream one packs it into /opt/, so the included one doesn't work.
Or should i fix it via sed...

Reply:
I don't understand what the problem is. Just move it where it should be.

Reply:
See my example for how I handled it.

Reply:
> See my example for how I handled it.

desktop-file-edit solved it